src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/model/Location.kt

```diff
@@ -11,4 +18,57 @@ data class Location(
 
   @Schema(description = "Flag indicates if the location is enabled, true is enabled and false is disabled.", example = "true")
   val enabled: Boolean,
+
+  @Schema(description = "Optional additional attributes about the room and its usage.", example = "true")
+  val extraAttributes: RoomAttributes? = null,
+)
+
+@Schema(description = "The additional attributes of a video location")
+data class RoomAttributes(
+  @Schema(description = "The status of the room (ACTIVE or INACTIVE)", example = "ACTIVE", required = true)
+  val locationStatus: LocationStatus,
+
+  @Schema(description = "An optional message relating to an inactive status", example = "Room damaged")
+  val statusMessage: String?,
+
+  @Schema(description = "The date the room is expected to be operational again", example = "2025-02-12")
+  val expectedActiveDate: LocalDate?,
+
+  @Schema(description = "The preferred usage for this room (COURT, PROBATION, SHARED, SCHEDULE)", example = "SHARED", required = true)
+  @Enumerated(EnumType.STRING)
```

Comment:
I don't think you need this `@Enumerated(EnumType.STRING)`

Reply:
Wouldn't it use the ordinal value for the enum without this?  I can try and take it off and see what the tests do.

Reply:
Ah - its the model. Needed on entity, not the response object. Done.